src/main/resources/templates/albums.html

```diff
@@ -49,9 +49,12 @@ <h1>My Current Albums </h1>
     <li th:text="'Song Count: '+ ${album.getSongCount}"></li>
     <li th:text="'Total time: '+ ${album.getLength}"></li>
     <img href=th:text=${album.getImageUrl} ></img>
-
+    <ol th:each="song: ${album.songs}">
+        <li th:if="${song.trackNumber}"></li>
```

Comment:
Here, you have your conditional logic controlling whether the list item displays, but you haven't told it what you want it to actually say. You'll need to add a `th:text` attribute with the song info.